graphql/end_to_end_test.go

```diff
@@ -93,14 +93,14 @@ func TestPathError(t *testing.T) {
 	val, err := e.Execute(context.Background(), builtSchema.Query, nil, q2)
 	assert.Nil(t, err)
 	assert.Equal(t, map[string]interface{}{
-		"inner": enumType(1),
+		"inner": "firstField",
```

Comment:
when you have a later commit that changes an earlier commit, please squash them together